tests/helpers/test_network.py

```diff
@@ -26,153 +26,145 @@ async def test_get_url_internal(hass: HomeAssistant):
         hass, {"internal_url": "https://192.168.0.1"},
     )
 
     assert hass.config.internal_url == "https://192.168.0.1"
-    assert _async_get_internal_url(hass) == "https://192.168.0.1"
-    assert (
-        _async_get_internal_url(hass, require_standard_port=True)
-        == "https://192.168.0.1"
-    )
-    assert _async_get_internal_url(hass, require_ssl=True) == "https://192.168.0.1"
+    assert _get_internal_url(hass) == "https://192.168.0.1"
+    assert _get_internal_url(hass, require_standard_port=True) == "https://192.168.0.1"
+    assert _get_internal_url(hass, require_ssl=True) == "https://192.168.0.1"
 
     with pytest.raises(NoURLAvailableError):
-        _async_get_internal_url(hass, allow_ip=False)
+        _get_internal_url(hass, allow_ip=False)
 
     # Test with internal URL: http://192.168.0.1:8123
     await async_process_ha_core_config(
         hass, {"internal_url": "http://192.168.0.1:8123"},
     )
 
     assert hass.config.internal_url == "http://192.168.0.1:8123"
-    assert _async_get_internal_url(hass) == "http://192.168.0.1:8123"
+    assert _get_internal_url(hass) == "http://192.168.0.1:8123"
 
     with pytest.raises(NoURLAvailableError):
-        _async_get_internal_url(hass, require_standard_port=True)
+        _get_internal_url(hass, require_standard_port=True)
 
     with pytest.raises(NoURLAvailableError):
-        _async_get_internal_url(hass, require_ssl=True)
+        _get_internal_url(hass, require_ssl=True)
 
     with pytest.raises(NoURLAvailableError):
-        _async_get_internal_url(hass, allow_ip=False)
+        _get_internal_url(hass, allow_ip=False)
 
 
 async def test_get_url_internal_fallback(hass: HomeAssistant):
     """Test getting an instance URL when the user has not set an internal URL."""
     assert hass.config.internal_url is None
 
     hass.config.api = Mock(
-        use_ssl=False, port=8123, base_url=None, local_ip="192.168.123.123"
+        use_ssl=False, port=8123, deprecated_base_url=None, local_ip="192.168.123.123"
     )
-    assert _async_get_internal_url(hass) == "http://192.168.123.123:8123"
+    assert _get_internal_url(hass) == "http://192.168.123.123:8123"
 
     with pytest.raises(NoURLAvailableError):
-        _async_get_internal_url(hass, allow_ip=False)
+        _get_internal_url(hass, allow_ip=False)
 
     with pytest.raises(NoURLAvailableError):
-        _async_get_internal_url(hass, require_standard_port=True)
+        _get_internal_url(hass, require_standard_port=True)
 
     with pytest.raises(NoURLAvailableError):
-        _async_get_internal_url(hass, require_ssl=True)
+        _get_internal_url(hass, require_ssl=True)
 
     hass.config.api = Mock(
-        use_ssl=False, port=80, base_url=None, local_ip="192.168.123.123"
+        use_ssl=False, port=80, deprecated_base_url=None, local_ip="192.168.123.123"
     )
-    assert _async_get_internal_url(hass) == "http://192.168.123.123"
+    assert _get_internal_url(hass) == "http://192.168.123.123"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)